Dockerfile

```diff
@@ -33,7 +33,8 @@ LABEL vendor=Sonatype \
 USER root
 
 # For testing
-RUN microdnf install procps
+RUN dnf update --allowerasing -y \
+&& dnf install -y procps
```

Comment:
this works, but there could be better approaches.

Reply:
> this works, but there could be better approaches.

I see `--allowerasing` seems like the essential part, and `microdnf` doesn't seem to support it whereas regular full blown `dnf` does